sdk/nextjs/src/pages/bucketing.ts

```diff
@@ -52,7 +53,7 @@ const bucketOrFetchConfig = async (
 
     return generateBucketedConfig({
         user,
-        config,
+        config: transformConfig(config),
```

Comment:
you should fix this at the fetching layer since the response is being asserted to be a ConfigBody, which it isn't